WordPress/WordPressTest/SiteSegmentsStepTests.swift

```diff
@@ -0,0 +1,45 @@
+import XCTest
+@testable import WordPress
+
+final class SiteSegmentsStepTests: XCTestCase {
+    private struct HeaderExpectations {
+        static let title = "Tell us what kind of site you'd like to make"
+        static let subtitle = "This helps us suggest a solid foundation. But you're never locked in -- all sites evolve!"
+    }
+
+    private var step: SiteSegmentsStep?
+
+    override func setUp() {
+        super.setUp()
+        step = SiteSegmentsStep(service: MockSiteSegmentsService())
```

Comment:
I'm seeing the following error when I attempt to run the tests:
```
.../Code/GitHub/WordPress-iOS/WordPress/WordPressTest/SiteSegmentsStepTests.swift:14:33: Missing argument for parameter 'builder' in call
```

It looks like the initializer now includes an additional parameter : `builder`.